redhat-cop/tool-integrations
# Add GitLab / ArgoCD bridge

### What does this PR do?
Adds a tool which periodically syncs private GitLab repositories in a group with an Argo CD instance.

### How should this be tested?
Deploy using Helm into an OpenShift project which also contains Argo CD (see README).

### Is there a relevant Issue open for this?
No

### Other Relevant info, PRs, etc.
~

### People to notify
cc: @redhat-cop/tool-integrations
